codespell_lib/data/dictionary.txt

```diff
@@ -7502,18 +7507,26 @@ couses->courses, causes,
 cousing->cousin
 couted->counted
 couter->counter
+coutermeasuere->countermeasure
+coutermeasueres->countermeasures
+coutermeasure->countermeasure
+coutermeasures->countermeasures
 couterpart->counterpart
+couters->counters
```

Comment:
Suggest routers and scouters, the latter is possibly a bit niche but hopefully okay
```suggestion
couters->counters, routers, scouters,
```

Reply:
Thanks